src/Lib.hs

```diff
@@ -44,17 +49,78 @@ startPetSelection user goldRemaining = do
 startBattle :: User -> IO ()
 startBattle user = do
     putStrBar
+    putStrLn "YOUR TEAM"
     putStrLn $ show (userRoster user)
-    putStrLn "\nBATTLE"
-    putStrLn "BATTLE"
-    putStrLn "BATTLE"
+    -- TODO add team name?
+
+    -- Create enemy team
+    -- TODO create their lineup using gold the same as the player
+    petOpponent1 <- getPet allPets
+    petOpponent2 <- getPet allPets
+    petOpponent3 <- getPet allPets
+    let opponentRoster = Roster {rosterPet1=(Just petOpponent1), rosterPet2=(Just petOpponent2), rosterPet3=(Just petOpponent3), rosterPet4=Nothing, rosterPet5=Nothing, rosterPet6=Nothing}
+    let userOpponent = User { userName="OPPONENT", userRoster=opponentRoster, userItemList=itemListEmpty }
+    putStrLn "\nOPPONENT TEAM"
+    putStrLn $ show (opponentRoster)
+
     -- TODO BATTLE
+    rosterBattle user userOpponent
+
+
+rosterBattle :: User -> User -> IO()
+rosterBattle user1 user2 = do
+  let r1pet = getRosterFirst (userRoster user1)
+  let r2pet = getRosterFirst (userRoster user2)
+
+  if isNothing r1pet
+    then putStrLn "user1 LOSES"
+    else putStrLn ""   -- wont compile without else?
```

Comment:
yells as a syntax error if i don't include the `else` is that a haskell thing?

Reply:
`if` in haskell requires a then and else clause, you can use `when` (https://hackage.haskell.org/package/base-4.16.2.0/docs/Control-Monad.html#v:when) instead